doc/basic_configuration.rst

```diff
@@ -55,17 +61,32 @@ In *all* build files::
     emails:
     - your_email@example.com
 
+Note that you need to have a local smtp service configured for email notifications.
+
+If you do not require email notifications, remove the configuration line entries.
```

Comment:
Even when you remove these global config settings some jobs will still send notification email to package specific email addresses.

Reply:
Thanks. I didn't know. I'll add a note.